Consistent-use-of-Image-component

vishwansh01 opened the pull request:
<!-- In order to keep off topic discussion to a minimum, it helps if the "work to be done" is already agreed on. -->
<!-- Ideally, a GitHub Issue with the label `Status: Consensus` will have been concluded, and linked to in this PR. -->
<!--
Thanks for submitting a pull request! Please provide enough information so that others can review your pull request.
-->

**What kind of change does this PR introduce?**
Changes img tags with Image tags in all .tsx files

<!-- E.g. a bugfix, feature, refactoring, etc… -->

**Issue Number:**
<!-- Pick one of the below options.  Please remove those which don't apply. -->
-  Closes #1885  <!-- Replace ___ with the issue number this PR resolves -->

**Screenshots/videos:**
<img width="526" height="568" alt="image" src="https://github.com/user-attachments/assets/432f9ab5-6e87-4011-b14a-e57030ed07ef" />

**Summary**
Relpaced all the img tag with Image tags in 
 - pages/index.page.tsx
 - pages/overview/pro-help/index.page.tsx
 - components/ui/AmbassadorsList.tsx

<!-- Explain the motivation for making this change. What existing problem does the pull request solve? -->
<!-- Try to link to an open issue for more information. -->

**Does this PR introduce a breaking change?**
No

<!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->

# Checklist

Please ensure the following tasks are completed before submitting this pull request.

- [x] Read, understood, and followed the [contributing guidelines](https://github.com/json-schema-org/website/blob/main/CONTRIBUTING.md).

vtushar06 commented:
Hii @vishwansh01, can you see please checkout https://github.com/json-schema-org/website/issues/1943, are things similar or can that issue be covered in this PR itself.
Please let me know.

vishwansh01 commented:
Hii @vtushar06 I looked at #1943 , it’s the same issue and can be folded into this PR. I’ll add the fixes here and push a follow up commit.

vishwansh01 commented:
modified:   components/Layout.tsx
 modified:   pages/blog/posts/[slug].page.tsx
 modified:   pages/community/index.page.tsx
 modified:   pages/index.page.tsx
 modified:  pages/blog/index.page.tsx
  modified:  cypress/components/Layout.cy.tsx

vishwansh01 commented:
Hi Utkarsh,
I have pulled the changes.
Could you please review it ?

vtushar06 commented:
Hii @vishwansh01, have you made changes also for issue #1943?

vishwansh01 commented:
Hii @vtushar06, I have made changes for issue #1943 .
Replaced all the CSS backgrounds using inline style={{ backgroundImage: ... }} with Image tag.
